demos/colab/distance_function.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
```

Comment:
this module should be deleted to avoid having multiple definitions of the distances already provided by norfair